### PR TITLE
Config debug features

### PR DESCRIPTION
**Pull request by p3jawors**
Pulled the desired features from `config_debug_rebase_util_removal`

- split up floating controller into `floating_task` and `floating_joint`
- added bell shaped dmp path planner
- added options to bell shaped and second order path planners to generate a path function of time
- added examples for all of the above, tested and work
- need to look into either configs or floating controller, as the sim arms move when under floating control
- travis ci checks pass

### CODE REVIEW

File: abr_control/controllers/path_planners/second_order_bell_shaped.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        self.state = state
+        self.dmps.reset_state()
+        self.dmps.goal = target - self.state
+
+
+    def step(self, error):
+        """
+        """
+        # get the next point in the target trajectory from the dmp
+        target = np.copy(self.dmps.step(error=error*0.65e1)[0])
```
Comment (studywolf) [inline abr_control/controllers/path_planners/second_order_bell_shaped.py]: what is this error gain term about? any idea? what happens if it's zero?

if i'm reading it right it just means that the DMP doesn't progress unless the error is pretty much zero.

**Comment by studywolf** [comment]
did some cleanup, there are a bunch of missing comments etc that still need to be filled in

File: abr_control/controllers/path_planners/second_order_bell_shaped.py (hunk 1 above)
Comment (p3jawors) [reply abr_control/controllers/path_planners/second_order_bell_shaped.py]: I'm not sure what the error gain term is, you wrote this up at some point to try on the arm iirc

**Comment by p3jawors** [comment]
Ready for review, already rebased to master
- Added missing docstrings
- reformatting of path planners to use more consistent naming between each other and controllers
- path planners always return pos and vel, and as separate parameters (no more stacking)
- have step and time based examples for both second order path planners
- travis ci checks passing

**Comment by studywolf** [comment]
this LGTM, but i'm not happy with the two different examples each for the `second_order` planners. not sure a better way to do it though, so let's just add a reminder to the backlog to address it

**Pull request merged**